silkworm/silkrpc/commands/ots_api.cpp

```diff
@@ -636,21 +636,18 @@ boost::asio::awaitable<void> OtsRpcApi::handle_ots_get_internal_operations(const
     co_return;
 }
 
-IssuanceDetails OtsRpcApi::get_issuance(const ChainConfig& chain_config, const silkworm::BlockWithHash& block) {
+IssuanceDetails OtsRpcApi::get_issuance(const ChainConfig& chain_config, const silkworm::BlockWithHash& ) {
     auto config = silkworm::ChainConfig::from_json(chain_config.config).value();
 
     if (config.protocol_rule_set != protocol::RuleSetType::kEthash) {
         return IssuanceDetails{};
     }
 
-    auto block_reward = protocol::EthashRuleSet::compute_reward(config, block.block);
```

Comment:
same as above

Reply:
After double check, the old code should be fine as it's only setting rewards when ethash is used. Change reverted.